.github/workflows/tests.yaml

```diff
@@ -269,7 +269,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["pypy-3.8", "pypy-3.9"]
+        python-version: ["pypy-3.9"]
```

Comment:
We should probably look into adding `pypy3.10`

Reply:
Surprise, it works! #9777